### DISCUSSION TITLE
x509: cannot validate certificate for <IP> because it doesn't contain any IP SANs

### DISCUSSION THREAD

**Discussion by divramod** [minio/minio#14559] [Q&A] [answered]
hey, 

i am testing the minio-operator within a kind kubernetes cluster.
i got everything running so far (installed the operator, created a tenant, ...) but now i am stuck configuring the cli client to connect to the tenant.

i can see, that the tenants service got a external ip over which i try to reach it:

![image](https://user-images.githubusercontent.com/1331662/158632677-596dab6d-2f97-4633-b38d-e7cb007ce448.png)

i use metallb as load balancer and i can reach the general console and the tenants console over my browser via the given external ips.

i tried to configure the client from my laptop (where kind is also running) with this command:

```sh
mc alias set minio https://172.19.255.104:443
```
this results in:
![image](https://user-images.githubusercontent.com/1331662/158633451-62c43d29-77c6-4b6d-81a6-744c64cad6e2.png)

i also read, that i should set ```export MINIO_SERVER_URL="https://172.19.255.104:443"``` but this has no effect.

can anyone help me get started here?

thx in advance

**Comment by vadmeste** [comment, marked as answer]
@divramod if you did not configure any TLS certificate while doing your setup, this means that the certificate was been generated automatically (self signed certificate) and it is normal that mc does not trust it.

You can either buy a TLS certificate or use --insecure flag when invoking mc command.
